odoo/addons/base/views/res_partner_views.xml

```diff
@@ -418,72 +394,37 @@
                                 </t>
                                 <t t-else="">
-                                    <div class="o_kanban_image_fill_left d-none d-md-block o_kanban_image_full">
-                                        <img t-attf-src="#{kanban_image('res.partner', 'avatar_128', record.id.raw_value)}" role="img" t-att-alt="record.id.value"/>
-                                    </div>
-                                    <div class="o_kanban_image d-md-none d-block o_kanban_image_full">
-                                        <img  t-attf-src="#{kanban_image('res.partner', 'avatar_128', record.id.raw_value)}" role="img" t-att-alt="record.id.value"/>
-                                    </div>
+                                    <field name="avatar_128" widget="image"/>
                                 </t>
-                                <div class="ribbon ribbon-top-right" invisible="active">
-                                    <span class="text-bg-danger">Archived</span>
-                                </div>
-                                <div class="oe_kanban_details d-flex flex-column justify-content-between">
-                                    <div>
-                                        <strong class="o_kanban_record_title oe_partner_heading"><field name="display_name"/></strong>
-                                        <div class="o_kanban_tags_section oe_kanban_partner_categories"/>
-                                        <ul>
-                                            <li t-if="record.parent_id.raw_value and !record.function.raw_value"><field name="parent_id"/></li>
-                                            <li t-elif="!record.parent_id.raw_value and record.function.raw_value"><field name="function"/></li>
-                                            <li t-elif="record.parent_id.raw_value and record.function.raw_value"><field name="function"/> at <field name="parent_id"/></li>
-                                            <li t-if="record.city.raw_value or record.country_id.raw_value">
-                                                <t t-if="record.city.raw_value"><field name="city"/><t t-if="record.country_id.raw_value">, </t></t>
-                                                <t t-if="record.country_id.raw_value"><field name="country_id"/></t>
-                                            </li>
-                                            <li t-if="record.email.raw_value" class="o_text_overflow"><field name="email"/></li>
-                                        </ul>
-                                    </div>
-                                    <div class="o_kanban_record_bottom">
-                                        <div class="oe_kanban_bottom_left"/>
-                                        <div class="oe_kanban_bottom_right"/>
+                            </aside>
+                            <main class="col-9 col-md-8 ps-2">
```

Comment:
... and this would give a better rendering on small screens.
```suggestion
                            <main class="col-9 col-sm-10 col-md-8 ps-2">
```

| Before | After |
|--------|--------|
| ![Capture d’écran 2024-06-12 à 10 06 28](https://github.com/odoo/odoo/assets/80679690/bb8a369a-a753-4141-81c5-3e1db0c50cd0) | ![Capture d’écran 2024-06-12 à 10 06 44](https://github.com/odoo/odoo/assets/80679690/21e04fe5-2990-4227-a2d6-51462cb2d696) |